backend/tests/Designer.Tests/Services/GiteaAPIWrapperTest.cs

```diff
@@ -355,23 +353,9 @@ public async Task Get_Repository()
             Assert.Equal(1769, result.Id);
         }
 
-        private static HttpContext GetHttpContextForTestUser(string userName)
+        private static GiteaAPIWrapper GetServiceForTest(HttpClient c)
```

Comment:
Might give this param a more readable name since we're already making changes here 😊 F.ex:
```suggestion
        private static GiteaAPIWrapper GetServiceForTest(HttpClient client)
```

Reply:
Completely agree! Fixed ✅